ui/components/app/transaction-decoding/components/ui/accreditation/accreditation.component.js

```diff
@@ -8,11 +8,9 @@ import {
 } from '../../../../../../selectors';
 import { I18nContext } from '../../../../../../contexts/i18n';
 
-import { TypographyVariant } from '../../../../../../helpers/constants/design-system';
-
 import Button from '../../../../../ui/button';
```

Comment:
Could use the Button from the component-library

Reply:
Hey @georgewrmarshall , 
👍Done.
Please look into this.